packages/design-system-react-native/.eslintrc.js

```diff
@@ -18,7 +18,9 @@ module.exports = {
     '**.config.js',
     '**.d.ts',
     '**.setup.js',
+    '**.scripts.ts',
     'svgMock.js',
+    'scripts/',
```

Comment:
non-blocking: I'm not sure we want to ignore all scripts from being linted but we can update this in a subsequent PR